packages/core/admin/admin/src/content-manager/components/RelationInput/RelationInput.js

```diff
@@ -34,20 +51,54 @@ const RelationInput = ({
   name,
   label,
   labelLoadMore,
-  listHeight,
   loadingMessage,
-  relations,
   onRelationAdd,
   onRelationLoadMore,
-  onSearchClose,
-  onSearchOpen,
   onRelationRemove,
+  onSearchClose,
   onSearchNextPage,
+  onSearchOpen,
   onSearch,
   placeholder,
   publicationStateTranslations,
+  relations,
   searchResults,
 }) => {
+  const listRef = useRef();
+  const [overflow, setOverflow] = useState('bottom');
+
+  // To clean (number relations + numberOfRelationsToDisplay as prop?)
+  const totalNumberOfRelations = relations.data?.pages[0]?.length;
+  const relationItemHeight = 58;
+  const numberOfRelationsToDisplay = 5;
```

Comment:
Same here.

Reply:
you mean create a constant out of component for `numberOfRelationsToDisplay`?
I was wondering if it would be passed as a prop, wdyt?

Reply:
If you prefer that we can do that too :)